### DISCUSSION TITLE
Create links under each user subdomain

### DISCUSSION THREAD

**Discussion by skeith** [YOURLS/YOURLS#3064] [Q&A] [answered]
Hi
Is it possible to make YOURLS create short links for each user under their own subdomain?

Example:
YOURLS installed on `link.yourls`
There are three users: `uno`, `bi`, and `hat`
for each user generated short links, will be created under their own subdomain:
```
uno.link.yourls/bu7dQ
bi.link.yourls/9kJHi
hat.link.yourls/P3d7pK
```

**Comment by ozh** [comment, marked as answer]
Virtually anything is possible :)

First option that comes to mind : simply have 3 installs under 3 domains. 
Second option : there's a "multiple user" plugin in the list, it may be an interesting start to code from
Third option : if all subdomains of link.yourls resolve to link.yourls, a simple plugin to modify the short URL when created is easy

In short : anything is possible. This situation will require something specific on your end, depending on your server setup.

**Reply by skeith** [reply]
Looks like option number 2 is the most promising (I did use a multiple user plugin). 
I made this question with the intention whether there's an already made solution available because there might be more than 100 users will be created. 

I'll make a feature request instead

**Comment by dzg** [comment]
Did you find or create a way to do this?

**Reply by skeith** [reply]
No I haven't. 
I forgot I asked about this.
